packages/cli/test/mocks/deployment.ts

```diff
@@ -211,7 +211,10 @@ function setupDeploymentEndpoints(): void {
   });
 
   client.scenario.get('/:version/deployments/:id/aliases', (req, res) => {
-    const limit = parseInt(req.query.limit);
+    const limit =
+      typeof req.query.limit === 'string'
+        ? parseInt(req.query.limit, 10)
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** prefer `Number` or `Number.parseInt` over `parseInt`‏

We could lint for this.